src/lib/helpers.ts

```diff
@@ -1 +1,13 @@
-export const isBrowser = () => typeof window !== 'undefined'
+import crossFetch from 'cross-fetch'
+
+type Fetch = typeof fetch
+
+export const resolveFetch = (customFetch?: Fetch): Fetch => {
+  if (customFetch) {
+    return customFetch
+  } else if (typeof fetch === 'undefined') {
+    return (crossFetch as unknown) as Fetch
+  } else {
+    return fetch
```

Comment:
I think we might have bind `this` properly and return fetch.bind(globalThis)

Reply:
Why do we need to bind?

Reply:
ah actually its fine here. If this helper method was part of an instance method of a class, it might have been an issue but here `this` will still point to the global object, so its fine.

Reply:
Just tested this and it is indeed problematic, since we call it like `await this.fetch(...)` and `this` is resolved at call time.